opentracing-metrics-prometheus-spring-autoconfigure/pom.xml

```diff
@@ -38,6 +38,12 @@
     </dependency>
 
     <dependency>
+      <groupId>io.opentracing.contrib</groupId>
+      <artifactId>opentracing-spring-web-autoconfigure</artifactId>
+      <scope>provided</scope>
```

Comment:
I think this depends on what we want, should be this integration noop or should it always work?